styles.css

```diff
@@ -245,49 +221,65 @@ input[type=checkbox] {
   vertical-align:middle;
   background-color: none;
   color: #f3f3f3;
-  text-align: center;
+  text-align: bottom;
   border-radius: 2px;
   border:1px solid #c3c3c3;
   margin-right: 8px;
-  margin-top: -3px;
+  margin-top: -10px;
 }
 input[type=checkbox]:checked + .chk:before {
   content: "\2713 ";
-  color: #303030;
+  color: rgba(0,0,0,0.5);
   font-size: 18px;
+  line-height: 0.5em;
 }
 input[type=checkbox]:checked + label{
   text-decoration: line-through;
 }
+aside{
```

Comment:
It may be better to assign a class to this aside, like 'author-box' or the like, because if you end up using multiple 'aside' elements on this page you will run into some issues.